sys-fs/bees/bees-0.6.3.ebuild

```diff
@@ -29,21 +29,11 @@ RDEPEND="${DEPEND}"
 CONFIG_CHECK="~BTRFS_FS"
 ERROR_BTRFS_FS="CONFIG_BTRFS_FS: bees does currently only work with btrfs"
 
-PATCHES=(
-	"${FILESDIR}/6001-lib-fix-non-local-lambda-expression-cannot-have-a-ca.patch"
```

Comment:
I did not yet check if 0.6.3 included these commits. But I didn't bump it because it caused errors during launch of bees and never managed to make progress. Is this still the case? Does 0.6.3 work for you?

Reply:
I haven't used bees much, but from what I can tell it's working fine for me. I checked and all the 600* patches have been merged into 0.6.3